writers/influxdb/README.md

```diff
@@ -68,10 +72,10 @@ and writer service, it contains [Grafana platform](https://grafana.com/) which c
 exploration and data visualization and analytics. In order to run all Mainflux core services, as well as mentioned optional ones, execute following command:
 
 ```bash
-docker-compose -f docker/docker-compose.yml -f docker/addons/influxdb/docker-compose.yml up -d
+docker-compose -f docker/addons/influxdb/docker-compose.yml up -d
```

Comment:
Shouldn't it be `docker-compose -f docker/addons/influxdb-writer/docker-compose.yml up -d`?